app/controllers/projects/samples/metadata/deletions_controller.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Projects
+  module Samples
+    module Metadata
+      # Controller actions for Project Samples Metadata Deletions Controller
+      class DeletionsController < Projects::Samples::ApplicationController
+        respond_to :turbo_stream
+
+        def new
+          authorize! @sample, to: :destroy_attachment?
```

Comment:
We probably want to authorize the project and to `update_sample?` since maintainer access level and higher should be allowed to delete metadata. `destroy_attachment?` doesn't make sense here since we are modifying metadata not the attachment

Reply:
Fixed in 969c18a281013c8d4311e2b0ddb355e3066185f6